.github/workflows/lychee-pr-check.yml

```diff
@@ -12,20 +12,23 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v4
-      - name: Get changed docs files
-        id: changed-md-files
-        run: |
-          CHANGED_FILES=$(git diff --name-only $GITHUB_SHA^ $GITHUB_SHA -- '*.md'  | paste -sd "," -)
-          echo "Changed Markdown files: '$CHANGED_FILES'"
-          echo "changed_files=${CHANGED_FILES:-}" >> "$GITHUB_OUTPUT"
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Get all changed markdown files
+        id: changed-markdown-files
+        uses: step-security/changed-files@3dbe17c78367e7d60f00d78ae6781a35be47b4a1 # v45.0.1
+        with:
+          # Avoid using single or double quotes for multiline patterns
+          files: |
+            **.md
+            docs/**.md
```

Comment:
Just out of curiousity, are files here not targetted by the glob above?

Reply:
actually, I think both of these may be wrong depending on what globbing system is in use. It should probably be just:
docs/**/*.md

Reply:
the only real way to check is to merge it and make some PRs...